amweekly/slack/jobs.py

```diff
@@ -42,15 +45,32 @@ def process_slash_command_webhook(webhook_transaction_id):
         return log
 
 
-def process_incoming_webhook(incoming_webhook_id):
-    try:
-        incoming_webhook = IncomingWebhook.objects.get(pk=incoming_webhook_id)
-        headers = {
-            'Content-type': 'application/json', }
+@job
+def process_incoming_webhook(start, end):
+    for incoming_webhook in IncomingWebhook.objects.all():
```

Comment:
I think this should be filtered for enabled `IncomingWebhook`s. Now that the crontab stuff isn't relevant, we can migrate the field onto the IncomingWebhook model